src/scss/grommet-core/_objects.world-map.scss

```diff
@@ -2,6 +2,12 @@
 
 .#{$grommet-namespace}world-map {
   width: 100%;
+
+  // IE11 fix world map height
+  @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+    padding-bottom: 49%;
```

Comment:
BTW this value comes from `height / width`, which is `460 / 940 = .489361702` rounding it off to `49`.

Reply:
Can you add a comment indicating that?